Makefile

```diff
@@ -15,7 +15,7 @@ ifdef CUSTOM_CA
 	cp -f ${CUSTOM_CA} docker/conf/certificates/
 endif
 
-up: update-box custom_ca
+up: clean update-box custom_ca
```

Comment:
Needed? If so, lets upstream this change.

Reply:
Not needed, was set while developing, forgot to remove